styles.css

```diff
@@ -432,3 +437,632 @@ footer {
 .footer-socials li {
   list-style: none;
 }
+
+/* Desktop version Media Query */
+
+@media screen and (min-width: 992px) and (max-width: 1399px) {
+  /* Toolbar Desktop */
+
+  .nav-bar h3 {
+    margin-left: 150px;
+  }
+
+  .nav-bar ul {
+    display: flex;
+    justify-content: space-between;
+    margin-right: 150px;
+  }
+
+  .nav-bar li {
+    margin-left: 40px;
+    list-style: none;
+  }
+
+  .link {
+    color: #fff;
+    text-decoration: none;
+    font-family: Inter, sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .fa-bars {
+    display: none;
```

Comment:
- [ ] Just as you mentioned, the hamburger icon is showing up in the desktop version. Try adding the `!important` to the display property. According to [developintelligence](https://www.developintelligence.com/blog/2018/02/css-important-rule-use-correctly/#:~:text=they%20are%20located.-,The%20!,certain%20design%20or%20layout%20goal.) _"The `!important` statement is used for breaking this order and adding weight to a specific declaration to put it in effect, ignoring the others. It is used for overriding the styles that are previously declared in other style sources"_

See guide:
```
.fa-bars {
    display: none !important;
}
```